specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json

```diff
@@ -0,0 +1,679 @@
+              "type": "string"
+            }
+          }
+        },
+        "name": {
+          "description": "name of the web site that had this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "The client request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "correlationRequestId": {
+          "description": "The correlation request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "address": {
+          "description": "HTTP request URL of this operation.",
+          "type": "string"
+        },
+        "verb": {
+          "description": "HTTP verb of this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "WebBackupOperationCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Web.BackupOperationCompleted event.",
+      "type": "object",
+      "properties": {
+        "appEventTypeDetail": {
+          "description": "Detail of action on the app",
+          "type": "object",
+          "properties": {
+            "action": {
+              "description": "Type of action of the operation.",
+              "type": "string"
+            }
+          }
+        },
+        "name": {
+          "description": "name of the web site that had this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "The client request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "correlationRequestId": {
+          "description": "The correlation request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the app service for the site API operation that triggered this event.",
+          "type": "string"
+        },
+        "address": {
+          "description": "HTTP request URL of this operation.",
+          "type": "string"
+        },
+        "verb": {
+          "description": "HTTP verb of this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "WebBackupOperationFailedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Web.BackupOperationFailed event.",
+      "type": "object",
+      "properties": {
+        "appEventTypeDetail": {
+          "description": "Detail of action on the app",
+          "type": "object",
+          "properties": {
+            "action": {
```

Comment:
Anonymous action class again (last one I will mention explicitly to avoid spam effect, but please change them all)